### ISSUE TITLE
Orphan in paragraph with `br`

### ISSUE THREAD

**Issue by doup** [rstacruz/unorphan#3]
I have a paragraph with an orphan, I think that it's maybe because of the `<br/>`. I suppose that the library thinks that `atemporal anonim` is on it's own row, so there nothing to unorphan. This is the HTML code:

```
<p>
    "La revolució és una força, una voluntat, un deure que és atemporal."<br>
    <em>Anò­nim</em>
</p>
```

And the result:

![orphan](https://cloud.githubusercontent.com/assets/188612/8443057/92e3ad7a-1f83-11e5-8cea-1ef3f763f73b.png)

It should be:

```
"La revolució és una força, una voluntat, un deure que
és atemporal."
Anò­nim
```

**Comment by rstacruz** [comment]
that's a little tricky. can you wrap the quote in a `<q>` or span or something?

**Comment by rstacruz** [comment]
Fixed in v1.2.0.
